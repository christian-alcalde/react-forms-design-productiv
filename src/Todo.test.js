import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

const testTodo = {
  id: "1",
  title: "test",
  description: "testDescription",
  priority: 1,
};

it("renders without crashing", function () {
  render(<Todo todo={testTodo} />);
});

it("matches snapshot", function () {
  const { container } = render(<Todo todo={testTodo} />);
  expect(container).toMatchSnapshot();
});
