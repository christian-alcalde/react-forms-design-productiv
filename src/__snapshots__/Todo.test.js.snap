// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        test
      </b>
       
      <small>
        (Priority:
        1
        )
      </small>
    </div>
    <div>
      <small>
        testDescription
      </small>
    </div>
  </div>
</div>
`;
